### ISSUE TITLE
Questions on Experiment Data Details

### ISSUE THREAD

**Issue by TingchenFu** [prateeky2806/ties-merging#3]
Thanks for your great work. I am writing to you because I have a question regarding the experiment details in your paper "Resolving Interference When Merging Models". Specifically,  I want to reimplement your experiment where you finetune a T5-base/T5-large/IA3 on PAWS dataset and WCS dataset before merging them with your proposed approach.

From the huggingface dataset https://huggingface.co/datasets/paws, I noticed that the PAWS dataset has several splits available, including Labeled (Final), Labeled (Swap-only), and Unlabeled (Final). Could you please clarify which split was used in your experiments? Do you include Labeled (Swap-only) and Unlabeled (Final) for training as well, or just use the Labeled (Final) subset for training and evaluation?

For the WCS dataset, it seems that it only contains a very small-scale test set with a 273 rows version and a 285 rows version (https://huggingface.co/datasets/winograd_wsc). I wonder how you fine-tune T5-base/T5-large on this dataset and which version of test set (273 rows or 285 rows) do you use?

**Comment by prateeky2806** [comment]
Hi Tingchen, 

Thanks for asking the question.

As mentioned in the [code here](https://github.com/prateeky2806/ties-merging/blob/27285bd1e73faf14da7a56527c4dd2fc19079b52/src/data/dataset_readers.py#L779-L782), for the paws dataset, we only use the "labeled_final" split for both training and testing. For more information, you can look at the dataset class [here](https://github.com/prateeky2806/ties-merging/blob/27285bd1e73faf14da7a56527c4dd2fc19079b52/src/data/dataset_readers.py#L15). 
The wsc dataset is part of superglue, we use the [wsc.fixed,](https://huggingface.co/datasets/super_glue/viewer/wsc.fixed/train) and we use the train split (554 rows) to FT the checkpoints and for merging we split the validation split (104) into val and test. The corresponding code can be found [here](https://github.com/prateeky2806/ties-merging/blob/27285bd1e73faf14da7a56527c4dd2fc19079b52/src/data/dataset_readers.py#L657).

Most of the dataset details are available in the code but if you can't find it then feel free to ping me. Let me know if there are other questions!

Thanks,
Prateek Yadav
